tests/functional/Ticket.php

```diff
@@ -4785,6 +4785,53 @@ public function testSurveyCreationOnReopened()
         $this->integer($it->count())->isEqualTo(1);
     }
 
+    public function testCronSurveyCreation()
```

Comment:
This test will pass even without proposed modifications. It should be rewritten to validates that it fails without any patch and that it is fixed with the proposed patch.